### DISCUSSION TITLE
Moving Panda Base

### DISCUSSION THREAD

**Discussion by branvu** [NVlabs/curobo#115] [Software Q&A]
Hello!

How would I move the base of the robot in the environment? I see the environment and robot are loaded separately. I noticed there is also a franka_mobile, though I don't want the additional 3 degrees of freedom but just want to set them permenantly. Also, is there a way to set the gripper to be closed or open before calling the motion_gen function? I'm interested in generating a trajectory to get to a grasping location in which the object will sit in between the grippers.

Thanks for any response or advice!

Brandon

**Comment by balakumar-s** [comment]
Regarding gripper closing can you check this example https://curobo.org/source/advanced_examples/2b_block_stacking_example.html ?

Regarding changing robot base pose, cuRobo represents the world w.r.t. robot base frame, so it shouldn't matter where you have the robot as long as you transform your goal target to be w.r.t. robot base frame.

**Reply by branvu** [reply]
Thanks this works
